integration-cli/docker_api_containers_test.go

```diff
@@ -432,7 +432,7 @@ func (s *DockerSuite) TestGetStoppedContainerStats(c *check.C) {
 	}()
 
 	// allow some time to send request and let daemon deal with it
-	time.Sleep(1 * time.Second)
+	time.Sleep(time.Second)
```

Comment:
I don't believe that this change makes something better :) It was easier to change it before.